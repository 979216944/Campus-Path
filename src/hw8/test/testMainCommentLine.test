# Verify that the program simply echos comment lines.

# one comment line

# two comment lines(1)
# two comment lines(2)

# quit
q