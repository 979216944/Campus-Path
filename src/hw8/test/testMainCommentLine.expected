Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # Verify that the program simply echos comment lines.

# one comment line

# two comment lines(1)
# two comment lines(2)

# quit
